waku/node/v2/rpc/wakurpc.nim

```diff
@@ -75,3 +75,17 @@ proc setupWakuRPC*(node: WakuNode, rpcsrv: RpcServer) =
 
     await node.query(HistoryQuery(topics: topics), handler)
     return true
+  
+  rpcsrv.rpc("waku_subscribe2") do(topic: string, contentFilters: seq[seq[string]]) -> bool:
+    debug "waku_subscribe"
+
+    # XXX: Hacky in-line handler
+    proc handler(msg: MessagePush) {.gcsafe, closure.} =
+      info "Hit subscribe response", nessages=msg.messages
+
+    var content = newSeq[ContentFilter]()
```

Comment:
confusing naming, this isn't content, it is a content filter. So contentFilter, filter, cf, or even foo would be better here

Reply:
also why is is it a seq but then it is passed into FilterRequest contentFilter? If this is plural, shouldn't it be contentFilters?

Indeed, https://specs.vac.dev/specs/waku/v2/waku-filter.html contentFilters